setup.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""
+Caring Caribou
+==============
+- A friendly automotive security exploration tool, initiated as part of the research project HEAVENS (HEAling Vulnerabilities to ENhance Software Security and Safety), now a stand-alone project.
+- A zero-knowledge tool that can be dropped onto an automotive network and collect information regarding what services and vulnerabilities exist.
+"""
+
+from setuptools import find_packages, setup
+
+version = "0.4dev"
+dl_version = "master" if "dev" in version else "v{}".format(version)
+
+print(r"""-----------------------------------
+ Installing Caring Caribou version {0}
+-----------------------------------
+""".format(version))
+
+setup(
+    name="caringcaribou",
+    version=version,
+    author="Kasper Karlsson",
+    author_email="hej@cc.py",
+    description="A friendly automotive security exploration tool",
+    long_description=__doc__,
+    keywords=["automotive", "security", "CAN", "automotive protocols", "fuzzing"],
+    url="https://cc.py",
```

Comment:
`url` is probably best set to https://github.com/CaringCaribou/caringcaribou/ as this is currently where the documentation can be found.